polkadot/node/subsystem-bench/src/subsystem-bench.rs

```diff
@@ -99,6 +99,10 @@ struct BenchCli {
 	/// Enable Cache Misses Profiling with Valgrind. Linux only, Valgrind must be in the PATH
 	pub cache_misses: bool,
 
+	#[clap(long, default_value_t = false)]
+	/// Shows the output in YAML format
+	pub ci: bool,
```

Comment:
Probably we can find a better name here like:
```
--output-format=yaml
--output-format=json
```

or even simpler
```
--yaml-output
```

Reply:
I think we can start with more obvious `--yaml-output`, because I have no plans for additional json one :-)